src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.left = '50%';
+spider.style.top = '50%';
```

Comment:
The spider image is not centered vertically based on the `wall` element, it's centered in the viewport. You should first get the `wall` element dimensions and then calculate the position of the spider based on that. 🕷️🧱